packages/mongo-nav/src/mongo-select/MongoSelect.spec.tsx

```diff
@@ -175,6 +172,40 @@ describe('packages/mongo-select', () => {
         expect(orgSettingsIcon).toBeNull();
       });
     });
+
+    describe('when a user filters organizations', () => {
```

Comment:
Need a test for when a user sets their `onChange` handler to make sure we don't filter.

Reply:
👍